src/TransactionTestsFiller/ttEIP2930/accessListStorageOver32BytesFiller.json

```diff
@@ -0,0 +1,31 @@
+{
+    "accessListStorageOver32Bytes" :
+    {
+        "expectException" :
+        {
+            ">=Frontier" : "2930AccessListStorageHashTooLong"
```

Comment:
Here the client should first check the byte of transaction type and reject it thus it will save time reading the structure for unsupported forks

Not crucial

Reply:
"TR_TypeNotSupported" error exception fails, compiler sees EIP bad input as priority.